androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/utils/CustomModifiers.kt

```diff
@@ -1,11 +1,24 @@
 package com.profusion.androidenhancedvideoplayer.utils
 
 import android.content.res.Configuration
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.remember
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
 
 fun Modifier.fillMaxSizeOnLandscape(orientation: Int) =
     when (orientation) {
         Configuration.ORIENTATION_LANDSCAPE -> this.then(Modifier.fillMaxSize())
         else -> this
     }
+
+fun Modifier.noRippleClickable(enabled: Boolean = true, onClick: () -> Unit): Modifier = composed {
+    clickable(
```

Comment:
I think this component is not necessary as it is only being used in one place. do u have another usage in mind?

Reply:
Nope, I don't, but I keep it to make it simpler to create a clickable without the ripple effect